src/graphql-client.ts

```diff
@@ -15,20 +15,13 @@ const link = new HttpLink({ uri, headers, fetch });
 
 export const client = new ApolloClient({ cache, link, defaultOptions: { query: { errorPolicy: "all" } } });
 
-type Mutation<T> = {
-    mutation: TypedDocumentNode<void, { input: T }>,
-    variables: { input: T },
-    asJson: () => unknown
-};

```

Comment:
Do you mind keeping the type?  I find it much clearer to have an understandable name for it.

Reply:
:+1: Okay if I reuse the existing Apollo-provided type?

Reply:
Sure